mesheryctl/internal/cli/root/system/system.go

```diff
@@ -16,6 +16,7 @@ package system
 
 import (
 	"errors"
+	"log"
```

Comment:
Just a small change @nitishm , can you replace log wilth logorus as we are using it on most of our code. 
Thanks :)

Reply:
Done. Amended my commit